tests/php/packages/logo/test_Logo.php

```diff
@@ -3,33 +3,18 @@
 use Automattic\Jetpack\Assets\Logo;
 
 class WP_Test_Logo extends WP_UnitTestCase {
+
+	/**
+	 * Ensure the rendered logo has all the CSS classes needed for styling.
+	 */
 	function test_constructor_default_logo() {
 		$logo = new Logo();
-		$expected = home_url( '/wp-content/plugins/jetpack/packages/logo/assets/logo.svg' );
-		$this->assertContains( $expected, $logo->render() );
-	}
-
-	function test_constructor_custom_logo() {
-		$example_logo = 'https://wordpress.com/logo.png';
-		$logo = new Logo( $example_logo );
-		$this->assertContains( $example_logo, $logo->render() );
+		$logo_render = $logo->render();
+		$this->assertContains( 'class="jetpack-logo"', $logo_render );
+		$this->assertContains( 'class="jetpack-logo__icon-circle"', $logo_render );
+		$this->assertContains( 'class="jetpack-logo__icon-triangle" points="15,19 7,19 15,3"', $logo_render );
+		$this->assertContains( 'class="jetpack-logo__icon-triangle" points="17,29 17,13 25,13"', $logo_render );
+		$this->assertContains( 'class="jetpack-logo__text"', $logo_render );
 	}
 
-	function test_render_img_tag() {
-		$logo = new Logo();
-		$output = $logo->render();
-		$url = home_url( '/wp-content/plugins/jetpack/packages/logo/assets/logo.svg' );
-
-		// Contains only a valid img tag.
-		$this->assertRegExp( '/^<img.*\/>$/', $output );
```

Comment:
Should we reuse this test but change it to be `<svg />` instead?

Reply:
Done